mayastor/src/rebuild.rs

```diff
@@ -29,13 +29,20 @@ pub enum RebuildError {
     TaskAlreadyExists { task: String },
     #[snafu(display("Missing rebuild destination {}", task))]
     MissingDestination { task: String },
+    #[snafu(display(
+        "{} operation failed because current rebuild state is {}.",
+        operation,
+        state,
+    ))]
+    OpError { operation: String, state: String },
```

Comment:
Should these be "Action" rather than operation since the trait is RebuildAction. Or maybe rename the RebuildAction to RebuildOperation?

Reply:
Done - changed RebuildAction to RebuildOperation